airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -120,6 +134,14 @@ def _save_blob_to_gcs(blob_to_save: storage.blob.Blob, file_path: str, disable_c
     return True
 
 
+def _delete_blob_from_gcs(blob_to_delete: storage.blob.Blob) -> bool:
+    """Deletes a blob from the bucket."""
+    print(f"Deleting {blob_to_delete.name}...")
```

Comment:
Should this be a log statement?

Reply:
This lib is always using `print`... It's 🆗  as it's mainly in `airbyte-ci`, which exposes the metadata lib stdout.